README.md

```diff
@@ -13,6 +13,11 @@ a place for contributed functionality for arkouda
 
 *For more details, visit [CONTRIBUTING.md](https://github.com/Bears-R-Us/arkouda-contrib/blob/main/CONTRIBUTING.md)*.
 
+## Basic Examples
+Basic examples of client-only and client-srever extensions to arkouda:
```

Comment:
Typo here: `client-srever` -> `client-server` and do we want to capitalize Arkouda or do you not feel strongly about that?